Sources/Player/Common/World/AsyncSchedulerFactoryProvider.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+// MARK: - AsyncSchedulerFactoryProvider.swift
+
+struct AsyncSchedulerFactoryProvider {
+	var newFactory: () -> AsyncSchedulerFactory
+}
+
+extension AsyncSchedulerFactoryProvider {
```

Comment:
[suggestion] not a big deal, but maybe we can follow others we have and split into 2 files, and keep the live one in `AsyncSchedulerFactoryProvider+Live`.

Reply:
I actually just saw that we don't separate any of the Live providers we use in `PlayerWorld` but we have to start somewhere :)

Reply:
That's a lie. We do on `DevelopmentFeatureFlagProvider` haha